s3router.js

```diff
@@ -25,9 +25,9 @@ function S3Router(options) {
 
     /**
      * Redirects image requests with a temporary signed URL, giving access
-     * to GET an image.
+     * to GET an upload.
      */
-    router.get(/\/img\/(.*)/, function(req, res) {
+    router.get(/\/upload\/(.*)/, function(req, res) {
```

Comment:
Should support both `/s3/img` and `/s3/upload` for backward-compatibility.

Also, what do you thinkg about pluralizing `upload` do be more REST-ful (avoiding verbs).  So `/s3/uploads`